core/collateral/engine.go

```diff
@@ -1898,14 +1898,15 @@ func (e *Engine) getLedgerEntries(ctx context.Context, req *types.TransferReques
 				return nil, err
 			}
 			for _, to = range ret.Balances {
-				lm = &types.LedgerEntry{
-					FromAccount: acc.ToDetails(),
-					ToAccount:   to.Account.ToDetails(),
-					Amount:      parts,
-					Type:        req.Type,
-					Timestamp:   now,
-				}
-				ret.Entries = append(ret.Entries, lm)
+				ret.Entries = append(ret.Entries, &types.LedgerEntry{
+					FromAccount:                    acc.ToDetails(),
+					ToAccount:                      to.Account.ToDetails(),
+					Amount:                         parts,
+					Type:                           req.Type,
+					Timestamp:                      now,
+					AccountFromPostTransferBalance: acc.Balance.Clone(),
```

Comment:
suggestion: as above it's named FromAccount, would it be more consistent to name these two: 
FromAccountPostTransferBalance and ToAccountPostTransferBalance also might as well get rid of the "PostTransferBalance" and just call it  FromAccountBalance and ToAccountBalance

Reply:
All right, which of these is better - AccountFrom or FromAccount?

Reply:
the one that is consistent with the rest of the universe, which I think is the latter.

Reply:
FromAccount is mentioned 45 times and AccountFrom 108 as Phil said, I'll go with it

Reply:
```
➜  vega git:(update_asset) grep -r FromAccount .|wc -l    
203
➜  vega git:(update_asset) grep -r AccountFrom .|wc -l    
53
```

Reply:
I don't think it really matters. We're not changing the whole universe now as this will be a massively breaking change. We just want to be consistent within the same object so making it [From|To]AccountBalance seems right. As it's also the more prevalent variant is even better :)

Reply:
All right its done
Everything related to ledger entries uses now `FromAccount`, `ToAccount`, except the DB columns - I havent changed them - they use the opposite - but their values are internally transferred in the query method in the ledger inside an object with proper naming